app.js

```diff
@@ -94,7 +94,7 @@ app.get('/debug/events', function (req, res) {
   res.send(`debug mode will log next ${Settings.debugEvents} events`)
 })
 
-const rclient = require('redis-sharelatex').createClient(
+const rclient = require('@overleaf/redis-wrapper').createClient(
```

Comment:
Looks like we've already required this once. Would this be cleaner?
```suggestion
const rclient = redis.createClient(
```

Reply:
Prettier will likely complain about this. Something for a follow-up.